src/services/formatting/smartIndenter.ts

```diff
@@ -464,9 +471,22 @@ namespace ts.formatting {
                 case SyntaxKind.GetAccessor:
                 case SyntaxKind.SetAccessor:
                     return child !== SyntaxKind.Block;
+                case SyntaxKind.ExportDeclaration:
+                    return child !== SyntaxKind.NamedExports;
                 default:
                     return false;
             }
         }
+
+        export function isIndentationPrevented(node: TextRangeWithKind) {
+            switch (node.kind) {
+                case SyntaxKind.NamedExports:
+                    return true;
+                // Allow indentation only when namedBindings is NamespaceImport.

```

Comment:
I later changed it to `// NamedImports has its own braces as Block does` `(so do not give it additional indentation)`